packages/rpc-methods/src/restricted/getLocale.ts

```diff
@@ -68,7 +68,7 @@ export const getLocaleBuilder = Object.freeze({
  */
 export function getImplementation({ getLocale }: GetLocaleMethodHooks) {
   return async function implementation(
-    _args: RestrictedMethodOptions<void>,
+    _args: RestrictedMethodOptions<any>,
```

Comment:
👀

Reply:
This one is a bit difficult, I'm not sure what we we should type this as since we don't use the arguments at all. Any was the best I could come up with, but I don't think it should matter much since the arguments are discarded.

Reply:
Doesn't work, but `RestrictedMethodParameters` does.